src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -309,21 +310,26 @@ bool LottieLoader::override(const char* slot)
 
     //parsing slot json
     LottieParser parser(temp, dirName);
-    auto sid = parser.sid();
+    auto sid = parser.sid(true);
+    
     if (!sid) {
         free(temp);
         return false;
     }
 
-    bool ret = false;
-    for (auto s = comp->slots.begin(); s < comp->slots.end(); ++s) {
-        if (strcmp((*s)->sid, sid)) continue;
-        ret = parser.parse(*s);
-        break;
-    }
+    int ret = 0, cnt = 0;
+
+    do {
```

Comment:
Thanks, and I changed a few lines:
- Remove `auto sid = parser.sid(true)` and sid check logic, so that while loop can start from first sid.
- Assure condition `idx == 0` to ensure only first loop can call `sid(true)`
- Add `if (idx < 1) success = false;`, in case the slot doesn't have any sid